frontend/components/GenerateInstallerModal/PlatformWrapper/_styles.scss

```diff
@@ -100,6 +94,12 @@
     }
   }
 
+  &__run-osquery-input {
+    .buttons {
+      top: 12px;
```

Comment:
Copy icon rendered at 40px on all other tabs, this one is a single line text area with no label so needs to be rendered at 12px to be in line with text area